test/max_workers_8.conf.in

```diff
@@ -0,0 +1,21 @@
+# NOTE: any changes here require changes to two other places
+# 1) tsl/test/postgresql.conf requires that it's prefix be the same as this file.
+# 2) appveyor.yml needs to be updated with the same options
+
+shared_preload_libraries=timescaledb
+max_worker_processes=16
```

Comment:
Same question

Reply:
In fact, I suggest we don't include any particular configuration options in the name of the file, since these options may change.

Reply:
See https://github.com/timescale/timescaledb/pull/4035#discussion_r796432569

Reply:
Still need a name for the configuration, but I'm open to suggestions. Since this configuration is intended for tests that has a limit of 8 background workers, it made more sense to use that in the name.